docs/timelines/timeline-options.md

```diff
@@ -0,0 +1,104 @@
+---
+id: timeline-options
+title: Timeline Options
+sidebar_label: Timeline options
+---
+
+Timelines have a number of options which set how they operate. 
+Press \<Options\> (or the \[Options\] softkey on the top level menu) then the **select** button for the timeline you wish to edit.
+
+--- 
+
+## Handle Tab
+
+All options are the same as for a cue, see [Options](../cues/playback-options.md#handle-tab).
+
+--- 
+
+## Times Tab
+
+![Timeline Options: Times](/docs/images/Timeline-Options-Times.png)
+
+These options let you set the duration and start time of the timeline. These set the loop points when you are using the 
+[Loop function](../timelines/timeline-options.md#timecode-tab) or are used to fire the timeline automatically when the timecode is in the correct range.
+They can also be used to set limits on the timeline so you can be sure nothing will happen if you get sent some wayward timecode.
+
+--- 
+
+## Fader Tab
+
+![Timeline Options: Fader](/docs/images/Timeline-Options-Fader.png)
+
+Sets what happens to an internal timecode source when the playback fader containing the timeline is raised or zeroed. If an external
+timecode source is being used then this option has no effect.
+
+Option              | Action
+--------------------|---
+Fader Raised Action | **No Action**: The internal timecode source is not changed (default).<br>Play: The internal timecode source is started.
+Fader Zero Action   | **No Action**: The internal timecode source is not changed (default).<br>Stop: The internal timecode source is stopped. <br>Pause: The internal timecode source is paused at the current time.
+
+> This option is useful for creating a complicated one-shot effect for busking. Create the effect as a timeline using internal timecode and set these options to **Play**/**Stop** - you can fire your effect just by raising the fader and when you lower the fader to zero the effect will reset.
+
+--- 
+
+## Release Tab
+
+![Timeline Options: Release](/docs/images/Timeline-Options-Release.png)
+
+Sets what happens to LTP attributes in playbacks which are killed by the timeline. This can be useful to make the output
+more predictable when you are skipping around the timeline, otherwise you can get confusing LTP values
+persisting from previous playbacks.
+
+Option              | Action
+--------------------|---
+Override Playback Release | **Off**: The playback's own release settings are used.<br>On: The Timeline-Global release settings override the playback settings.
+Release Playbacks to Home | Off: LTP attributes are left when when killing playbacks.<br>**On**: Attributes will return to home if no previous LTP values are set.
```

Comment:
```suggestion
Override Playback Release | **Off**: The playback's own release settings are used (default).<br>On: The Timeline-Global release settings override the playback settings.
Release Playbacks to Home | Off: LTP attributes are left when when releasing the last playback controlling an attribute. <br>**On**: Attributes will return to home if no previous LTP values are set (default).
```

Reply:
Also double `when`

Reply:
Default settings are indicated by the bold **Off** or **On**, I'm not putting in (default) every time

Reply:
It was more for consitency - sometimes there was `(default)` (see Fader Raised / Zero). Don't really mind either way tbh

Reply:
Ah I had not noticed that... there is a mixture, unfortunately, as we move from having option lists in a big paragraph of text to the clearer way of showing them in a table, I am hoping to use the standard of showing default option as bold and not needing to say "default".